src/go/rpk/pkg/cli/cmd/cluster/license/info.go

```diff
@@ -43,10 +44,13 @@ func newInfoCommand(fs afero.Fs) *cobra.Command {
 			if info.Properties != (admin.LicenseProperties{}) {
 				expired := info.Properties.Expires < 0
 				if format == "json" {
+					tm := time.Unix(info.Properties.Expires, 0).Format("Jan 2 2006")
 					props, err := json.MarshalIndent(struct {
-						admin.LicenseProperties
-						Expired bool `json:"license_expired,omitempty"`
-					}{info.Properties, expired}, "", "  ")
+						Organization string
+						Type         string
+						Expires      string
```

Comment:
Can we add the json struct tags are missing. This will look like:

```json
{
  "Organization": "redpanda",
  "Type": "free_trial",
  "Expires": "Jul 11 2122",
  "license_expired": true
}
```

Also, version is missing. I think the ducktape test will need to change.

Reply:
@twmb : I think we should be consistent with the plain text format so

```json
{
    "version" : 1
    "organization": "redpanda",
    "type": "free_trial",
    "expires": "Jul 11 2122",
    "license_expired": true
}
```

Instead of current:

```json
{
    "expires": 34
    "format_version": 0,
    "org": "redpanda-testing",
    "type": "enterprise"
}
```

wdyt?

Reply:
Version is an implementation detail that the user shouldn't see

Reply:
and nice catch i'll need to modify the ducktape test to be "Jul 11 2122"

Reply:
Sorry, can you change:
```
	Organization string
	Type         string
	Expires      string
```
for
```go
	Organization string `json:"organization,omitempty"`
	Type         string `json:"type,omitempty"`
	Expires      string `json:"expires,omitempty"`
```

So we have a uniform output with plain text vs JSON vs JSON with an expired license?

This will also require a change in the expected_license variable in ducktape test.

Reply:
I will modify this in a follow up PR, to get CI passing with modifications of the env vars in CI (and to not affect others) we decided to merge this now